qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -67,29 +67,28 @@ def __init__(
     ) -> None:
         """
         Args:
-            quantum_kernel: a quantum kernel to be used for classification. Has to be ``None`` when
-                a precomputed kernel is used.
+            quantum_kernel: A quantum kernel to be used for classification.
+                Has to be ``None`` when a precomputed kernel is used. If None,
+                and ``precomputed``is ``False``, the quantum kernel will default to
```

Comment:
```suggestion
                and ``precomputed`` is ``False``, the quantum kernel will default to
```
Missing a space

![image](https://github.com/qiskit-community/qiskit-machine-learning/assets/40241007/a539abc0-d39c-48fe-a6a4-716ff4a98f9c)

Reply:
Thanks!